projects/origin-release/golang-1.10/Dockerfile

```diff
@@ -26,9 +26,19 @@ RUN yum install -y epel-release && \
 
 ADD *.rpm /tmp/origin-rpm/
 RUN yum install -y /tmp/origin-rpm/*.rpm && \
-    yum clean all && \
-    curl https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
-    go get golang.org/x/tools/cmd/cover \
+    yum clean all 
+
+#Download go for amd64 unless other archs are detected
```

Comment:
Would we rather see `Dockerfile.ppc64le` than `case` statements on every point of variation between image manifests?

Reply:
Yes

Reply:
Each architecture should have its own docker file

Reply:
This implies that (in the future) amd64 should have its own Dockerfile. 
Long term, I think its better to maintain a Dockerfile per OS base (rhel/centos and soon fedora). If there are arch specific dependencies, they should be addressed at the repo level first, and if that isn't possible (like this case) then logically either in the dockerfile or as a parameter passed at build time.
FWIW, the origin, web-console-server, and image-registry Dockerfile's work as is, so I don't expect this logic to be in a lot of places.

I think the larger issue here is that there hasn't been a roadmap discussed for how these 'other' archs are to be integrated, and manifest listed.